manim/mobject/types/image_mobject.py

```diff
@@ -44,6 +43,9 @@ def reset_points(self):
         )
         self.center()
         h, w = self.get_pixel_array().shape[:2]
+        resolution_of_final_video = 1080
```

Comment:
this line should be removed, and instead, this variable should come from ImageMobject